incubator/patroni/templates/NOTES.txt

```diff
@@ -0,0 +1,29 @@
+Getting Started:
```

Comment:
I know some other charts have this, but it looks weird given the heading is "Notes:" already, so I think we should remove this line.